src/Components/PlayBar/Components/ControllerBox/ControllerBox.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faRandom, faRetweet, faPlay, faStepForward, faStepBackward } from "@fortawesome/free-solid-svg-icons/";
+import styled from "styled-components";
+
+function ControllerBox({ width, position, background_color }) {
+  return (
+    <ControllerBoxd width={width} position={position} background_color={background_color}>
```

Comment:
```suggestion
    <ControllerBox width={width} position={position} background_color={background_color}>
```
반복적으로 보이는데 오타인가요? 의도인가요?

Reply:
수정하였습니다~

Reply:
수정하였습니다 ~